packages/mdc-floating-label/_mixins.scss

```diff
@@ -19,13 +19,11 @@
 // Public
 
 @mixin mdc-floating-label-color($color) {
-  .mdc-floating-label {
-    @include mdc-theme-prop(color, $color);
-  }
+  @include mdc-theme-prop(color, $color);
 }
 
-@mixin mdc-floating-label-invalid-shake-keyframes($modifier, $positionY, $positionX: 0%, $scale: .75) {
-  @keyframes invalid-shake-float-above-#{$modifier} {
+@mixin mdc-floating-label-shake-keyframes($modifier, $positionY, $positionX: 0%, $scale: .75) {
+  @keyframes shake-float-above-#{$modifier} {
```

Comment:
CSS `@keyframes` are globally-scoped, so we need to prefix all of their names with `mdc-floating-label-` to avoid collisions.